redundancy/redundancy.go

```diff
@@ -100,16 +111,16 @@ func (nr *nodeRedundancy) ResetInactivityIfNeeded(selfPubKey string, consensusMs
 	}
 
 	nr.mutNodeRedundancy.Lock()
-	nr.roundsOfInactivity = 0
+	nr.handler.ResetRoundsOfInactivity()
 	nr.mutNodeRedundancy.Unlock()
 }
 
 func (nr *nodeRedundancy) isMainMachineActive() bool {
-	if nr.redundancyLevel < 0 {
+	if !nr.handler.IsRedundancyNode(nr.maxRoundsOfInactivity) {
```

Comment:
I think that this condition could be deleted if the change suggested above in redundancyHandler.go, regarding the condition **<=** instead **<** will be done! We also can keep it like this, if we want that this method always returns true for the main machine, regardless the number of missed rounds by itself

Reply:
will refactor